Ports/optipng/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="optipng"
+version='0.7.7'
+files="http://downloads.sourceforge.net/optipng/optipng-${version}.tar.gz optipng-${version}.tar.gz 4f32f233cef870b3f95d3ad6428bfe4224ef34908f1b42b0badf858216654452"
+auth_type=sha256
+useconfigure='true'
+
+configure() {
+	run ./configure
+}
```

Comment:
is this not what the default `configure` does?

Reply:
Default configure adds `--host=...`. Maybe we should look into making that optional in the future or move it to `configopts` for consistency.